conf/logback.xml

```diff
@@ -23,8 +23,18 @@
     </encoder>
 </appender>
 
+<appender name="Sentry" class="io.sentry.logback.SentryAppender">
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+        <level>ERROR</level>
+    </filter>
+    <filter class="monitoring.PiiFilter">
```

Comment:
Might be worth adding a comment to this code...

https://github.com/guardian/support-frontend/blob/16b54b1f7d02481f387ce09b38c68628cafb800a/app/monitoring/Logging.scala#L10-L13

... to explain that the filter is referenced in logback.xml.

Reply:
That would be a useful comment. Will do.

Reply:
Alernatively you could define the filter itself here in logback.xml i.e. the opposite filter to 

https://github.com/guardian/support-frontend/blob/master/conf/logback.xml#L7-L12